Lesson2Xcode/ViewControllers/ComponentsViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ComponentsViewController.swift
+//  Lesson2Xcode
+//
+//  Created by Алсу Хайруллина on 20.09.2021.
+//
+
+import UIKit
+
+class ComponentsViewController: UIViewController, UIPickerViewDataSource, UIPickerViewDelegate {
+
+    @IBOutlet weak var testLabel: UILabel!
+    @IBOutlet weak var testSwitch: UISwitch!
+    @IBOutlet weak var testSlider: UISlider!
+    @IBOutlet weak var testProgressView: UIProgressView!
+    
+    @IBOutlet weak var pickerView: UIPickerView!
+    @IBOutlet weak var segmentControl: UISegmentedControl!
+    
+    let choices = ["Linux","MacOS","Windows"]
+    
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        return choices[row]
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        return choices.count
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    
+    }
+    
+    @IBAction func backButton(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func changeButton(_ sender: Any) {
```

Comment:
```suggestion
    @IBAction func didTapChangeButton(_ sender: Any) {
```
навзвание функции звучит как проперти